docs/getting-started/configure.md

```diff
@@ -256,6 +256,8 @@ additional_packages = ["git", "make", "vim"]
 Edit the `config/redhat.pkrvars.hcl` file to configure the credentials for your Red Hat Subscription
 Manager account.
 
+You can also disable Red Hat Subscription Manger by setting `rhsm_enabled = false`.
```

Comment:
There were additions for other options that are not mentioned here.

Reply:
added